packages/components/src/modelLoader.ts

```diff
@@ -0,0 +1,45 @@
+import { INodeOptionsValue } from './Interface'
+import axios from 'axios'
+
+/* This is a temp URL, will be replaced with the actual URL from the main repo */
+const MASTER_MODEL_LIST =
+    'https://raw.githubusercontent.com/vinodkiran/Flowise/FEATURE/externalize-model-list/packages/components/models.json'
+
+export enum MODEL_TYPE {
+    CHAT = 'chat',
+    LLM = 'llm',
+    EMBEDDING = 'embedding'
+}
+
+const getModelConfig = async (category: MODEL_TYPE, name: string) => {
+    const modelFile = process.env.MODEL_LIST_CONFIG_JSON || MASTER_MODEL_LIST
```

Comment:
we need to add `MODEL_LIST_CONFIG_JSON` to other places like `start.ts` and oother `.env.example` files

Reply:
Done!